plugins/utils/commands/install-gitleaks.md

```diff
@@ -0,0 +1,210 @@
+---
+description: Install gitleaks for detecting secrets in code
+---
+
+## Name
+utils:install-gitleaks
+
+## Synopsis
+```
+/utils:install-gitleaks
+```
+
+## Description
+The `utils:install-gitleaks` command installs [gitleaks](https://github.com/gitleaks/gitleaks), a SAST tool for detecting and preventing hardcoded secrets like passwords, API keys, and tokens in git repositories and files.
+
+Gitleaks is essential for verifying that sensitive information has been properly redacted before committing code or sharing files.
+
+## Implementation
+
+### Step 1: Check if Gitleaks is Already Installed
+
+```bash
+which gitleaks
+```
+
+**Expected outcomes:**
+- **Path returned** (e.g., `/usr/local/bin/gitleaks`): Gitleaks is already installed
+  - Check version: `gitleaks version`
+  - If version is recent (8.0+), skip to Step 4
+  - If version is old, proceed to Step 2
+- **Command not found**: Proceed to Step 2
+
+### Step 2: Detect Operating System and Architecture
+
+Determine the user's platform to download the correct binary:
+
+```bash
+uname -s
+uname -m
+```
+
+**Common platform mappings:**
+- `Darwin` + `x86_64` → macOS Intel (`darwin_x64`)
+- `Darwin` + `arm64` → macOS Apple Silicon (`darwin_arm64`)
+- `Linux` + `x86_64` → Linux AMD64 (`linux_x64`)
+- `Linux` + `aarch64` → Linux ARM64 (`linux_arm64`)
+
+### Step 3: Install Gitleaks
+
+Choose the appropriate installation method based on the platform:
+
+#### Option A: macOS (using Homebrew - Recommended)
+
+```bash
+# Check if Homebrew is installed
+which brew
+
+# If Homebrew is available:
+brew install gitleaks
+```
+
+#### Option B: macOS (manual installation)
+
+```bash
+# Determine architecture
+ARCH=$(uname -m)
+if [ "$ARCH" = "arm64" ]; then
+    PLATFORM="darwin_arm64"
+else
+    PLATFORM="darwin_x64"
+fi
+
+# Download latest release
+VERSION="8.20.1"  # Check https://github.com/gitleaks/gitleaks/releases for latest
+curl -sSfL "https://github.com/gitleaks/gitleaks/releases/download/v${VERSION}/gitleaks_${VERSION}_${PLATFORM}.tar.gz" -o gitleaks.tar.gz
+
+# Extract and install
+tar -xzf gitleaks.tar.gz gitleaks
+sudo mv gitleaks /usr/local/bin/gitleaks
+sudo chmod +x /usr/local/bin/gitleaks
+
+# Clean up
+rm gitleaks.tar.gz
+```
+
+#### Option C: Linux (manual installation)
```

Comment:
I think option C for linux should be that if it is fedora it uses the package https://koji.fedoraproject.org/koji/packageinfo?packageID=35909